todoapp/src/App.tsx

```diff
@@ -2,27 +2,39 @@ import "./App.css";
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import Home from "./components/Home";
 import Navbar from "./components/Navbar";
+// import AddTask from "./components/AddTask";
+import NotFound from "./components/Not-found";
+import Taskdetails from "./components/Taskdetails";
+import CreateTask from "./components/CreateTask";
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import {
+  QueryClient,
+  QueryClientProvider,
+  useQuery,
+} from "@tanstack/react-query";
+
+const queryClient = new QueryClient();
 
 function App() {
   return (
-    <div className="contents">
-      <BrowserRouter>
-        <Navbar/>
-        <Routes>
-          <Route path="/" element={<Home />}>            
-          </Route>
-          {/* <Route path="/create">
-              <Create />
-            </Route>
-            <Route path="/blogs/:id">
-              <BlogDetails />
-            </Route>
-            <Route path="*">
-              <NotFound />
-            </Route> */}
-        </Routes>
-      </BrowserRouter>
-    </div>
+    <QueryClientProvider client={queryClient}>
+      <div className="contents">
+        <BrowserRouter>
+          <Navbar />
+          <Routes>
+            <Route path="/" element={<Home />}></Route>
+
+            <Route path="/create" element={<CreateTask />}></Route>
```

Comment:
/create Or /task/:id 

can we keep one route constants and fetch it from there?